### PR TITLE
app/fear-and-greed

### PR DESCRIPTION
**Pull request by IAmKio**
## Description
This app serves as an example app to show other developers what they can build, using various libraries that will ship with PillarX.

New developer toolchain libraries include:

UI frameworks / libraries: 
- `@emotion/react`
- `@emotion/styled`
- `@mui/joy`
- `@react-spring/web`
- `react-google-charts`
- `webfontloader`

HTTP libraries
- `axios`

Screenshot:
PillarX - FAG

### CODE REVIEW

File: src/containers/Main.tsx [hunk 1]
```diff
@@ -97,9 +97,9 @@ const Main = () => {
 }
 
 const AuthContentWrapper = styled.div`
-  max-width: 500px;
+  // max-width: 500px;
   margin: 0 auto;
-  padding: 60px 20px 120px;
+  // padding: 60px 20px 120px;
```
Comment (IAmKio) [inline src/containers/Main.tsx:102]: Commented these lines out for now - we'll figure out a permanent solution for this

**Review by taylorferran — APPROVED** [review]
Formatting on the main page looks weird to me now, apart from that LGTM

File: .gitignore [hunk 2]
```diff
@@ -27,3 +27,4 @@ yarn-error.log*
 
 # Sentry Config File
 .sentryclirc
+.env
```
Comment (poocart) [inline .gitignore:30]: can we move it to `# misc`where the rest of `.env.*`are? nothing crucial, just tidier heh

File: .eslintrc.json [hunk 3]
```diff
@@ -39,6 +39,7 @@
         "import/no-named-as-default-member": "off",
         "import/no-named-as-default": "off",
         "no-console": ["error", { "allow": ["warn", "error"] }],
-        "quotes": ["error", "single"]
+        "quotes": ["error", "single"],
+        "react/react-in-jsx-scope": "off"
```
Comment (poocart) [inline .eslintrc.json:43]: can we add this as `"plugin:react/jsx-runtime"` extension as recommended on rule doc (see last paragraph) https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md

File: src/apps/fear-and-greed/index.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+ * for API data
+ */
+interface ILatestData {
+  value: string;
+  value_classification: string;
+  timestamp: string;
+  time_until_update: string;
+}
+
+export const App = () => {
```
Comment (poocart) [inline src/apps/fear-and-greed/index.tsx:39]: no need to export as module as it won't be used, default export at the very bottom is the one used

File: src/apps/fear-and-greed/index.tsx [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+  const [fagIndex, setFagIndex] = React.useState<number>(0);
+
+  /**
+   * Fire this once, on load
+   */
+  useEffect(() => {
+    /**
+     * Fetch the data from our API
+     */
+    axios.get('https://api.alternative.me/fng/?limit=1')
```
Comment (poocart) [inline src/apps/fear-and-greed/index.tsx:91]: requires a thought process how we can avoid malicious APIs to enter the arena

File: src/components/BottomMenu/index.tsx [hunk 6]
```diff
@@ -111,7 +110,7 @@ const MenuItem = styled.div`
 `;
 
 const Wrapper = styled.div`
-  max-width: 800px;
+  // max-width: 800px;
```
Comment (poocart) [inline src/components/BottomMenu/index.tsx:113]:
```suggestion
  // max-width: 800px; TODO: inspect and remove/adjust
```

File: src/containers/Main.tsx [hunk 7]
```diff
@@ -97,9 +97,9 @@ const Main = () => {
 }
 
 const AuthContentWrapper = styled.div`
-  max-width: 500px;
+  // max-width: 500px;
```
Comment (poocart) [inline src/containers/Main.tsx:100]:
```suggestion
  // max-width: 500px; TODO: inspect and remove/adjust
```

File: src/containers/Main.tsx (hunk 1 above)
Comment (poocart) [inline src/containers/Main.tsx:102]:
```suggestion
  // padding: 60px 20px 120px; TODO: inspect and remove/adjust
```

File: src/apps/fear-and-greed/index.tsx (hunk 4 above)
Comment (IAmKio) [reply src/apps/fear-and-greed/index.tsx:39]: oops didn't notice that export

**Pull request merged**